staging/maven-plugin/README.md

```diff
@@ -14,7 +14,7 @@
 -->
 # Nexus Staging Maven Plugin
 
-Maven Plugin to control Nexus Staging workflow. While the maven plugin ("staging client") part is OSS, to use staging features it need a Sonatype Nexus Professional instance 2.1+ on the server side!
+Maven Plugin to control [Nexus Staging](https://help.sonatype.com/repomanager2/staging-releases/staging-overview) workflow. While the maven plugin ("staging client") part is OSS, to use staging features it need a Sonatype Nexus Professional instance 2.x on the server side!
```

Comment:
Maven Plugin to control [Nexus Staging](https://help.sonatype.com/repomanager2/staging-releases/staging-overview) workflow. While the Maven Plugin ("staging client") part is OSS, you must have a Sonatype Nexus Professional 2.x instance on the server side to use staging features.

Reply:
@lisadurant reworded as proposed in [117ccb0](https://github.com/sonatype/nexus-maven-plugins/pull/94/commits/117ccb0f0167a5c34e0c4452b8cc3518004f884b)